web/gds-admin-ui/src/pages/account/Login.js

```diff
@@ -7,19 +7,50 @@ import AccountLayout from './AccountLayout';
 import SignWithGoogle from '../../components/Auth/SignWithGoogle';
 import { loginUser } from '../../redux/auth/actions';
 import config from "../../config";
+import { APICore, setCookie } from '../../helpers/api/apiCore';
+import { getCookie } from '../../utils';
 
+const api = new APICore()
 
 const Login = (): React$Element<any> => {
     const dispatch = useDispatch();
     const { userIsLoggedIn, user } = useSelector((state) => ({
         userIsLoggedIn: state.Auth.userIsLoggedIn,
         user: state.Auth.user
     }))
+    const [authenticated, setAuthenticated] = React.useState(false)
```

Comment:
`authenticated` is not really the right verb here; I'd prefer to use `protected`, `csrfProtected`, or `doubleCookieProtected` for this state; otherwise folks who are reading this code will be confused about what is happening with the `GET` request.

Reply:
Yes, I had some doubt about it